DetectorDescription/DDCMS/interface/DDCMSPlugins.h

```diff
@@ -0,0 +1,68 @@
+//==========================================================================
+//  AIDA Detector description implementation 
+//--------------------------------------------------------------------------
+// Copyright (C) Organisation europeenne pour la Recherche nucleaire (CERN)
+// All rights reserved.
+//
+// For the licensing terms see $DD4hepINSTALL/LICENSE.
+// For the list of contributors see $DD4hepINSTALL/doc/CREDITS.
+//
+// Author     : M.Frank
+//
+//==========================================================================
+//
+// DDCMS is a detector description convention developed by the CMS experiment.
+//
+//==========================================================================
+#ifndef DD4HEP_DDCMS_DDCMSPLUGINS_H
+#define DD4HEP_DDCMS_DDCMSPLUGINS_H
+
+// Framework includes
+#include "DetectorDescription/DDCMS/interface/DDCMS.h"
+#include "DD4hep/Plugins.h"
+#include "CLHEP/Units/SystemOfUnits.h"
```

Comment:
can we get rid of this?
I though we decided to obsolete CLHEP. No new code should use it anymore!